benchmarks/src/main/scala/walkfiletree/generate.py

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
If we had a `prepare()` method in `benchmarks.Benchmark` we could put the directory structure generation there.
And have cleanup in `cleanup()` method. 

And/or just create these files inside the tmp dir.

Reply:
Yeah, I wasn't sure what direction you guys would have wanted, so I tried to keep it simple for this initial patch with the checked in generated dummy directory structure.

I'll look into enhancing the Benchmark trait.